orp/WorldObject[] result
string method
